vector/src/main/java/im/vector/util/PreferencesManager.java

```diff
@@ -377,6 +378,8 @@ public static boolean useMatrixApps(Context context) {
         return PreferenceManager.getDefaultSharedPreferences(context).getBoolean(SETTINGS_USE_MATRIX_APPS_PREFERENCE_KEY, false);
     }
 
+
+    // true par defaut
```

Comment:
This comment should be part of the /** */ section.

Reply:
Ok